assets/react/controllers/Incident/ActionBox.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from "react";
+import { Button, Divider, Drawer, Popconfirm, Modal, Form } from "antd";
+import RescheduleTask from "./ActionBox/RescheduleTask";
+import ApplyPriceDiffTask from "./ActionBox/ApplyPriceDiffTask";
+import TransporterReport from "./ActionBox/TransporterReport";
+
+import { useTranslation } from "react-i18next";
+
+import store from "./incidentStore";
+
+async function _handleCancelButton(id) {
+  const httpClient = new window._auth.httpClient();
+  return await httpClient.put(
+    window.Routing.generate("api_incidents_action_item", { id }),
+    { action: "cancel_task" },
+  );
+}
+
+const styles = {
+  btn: {
+    width: "100%",
+  },
+};
+
+export default function () {
+  const { loaded, incident, order, images, transporterEnabled } =
```

Comment:
Also, it's better to read from the store using selectors, it makes it easier to refactor store state shape